base/sysimg.jl

```diff
@@ -180,6 +180,27 @@ big(n::Integer) = convert(BigInt,n)
 big(x::FloatingPoint) = convert(BigFloat,x)
 big(q::Rational) = big(num(q))//big(den(q))
 
+const _fact_table128 =
```

Comment:
There are cross dependencies that break if the entire file is shifted. These were the only `UInt128` dependent code that was there. I thought of putting these in an `int128funcs.jl` file, but this seemed too small to go into a separate file.